src/PKSim.Core/Services/ChartsDataCreator.cs

```diff
@@ -205,8 +207,8 @@ protected IEnumerable<DataRow> RowsForDataField(string fieldName)
 
       private CurveData<TX, TY> getOrCreateCurve(PaneData<TX, TY> pane, IReadOnlyDictionary<string, string> seriesFieldValues, INumericValueField yAxisField)
       {
-         string curveCaption = captionFor(seriesFieldValues.Values);
-         string curveId = idFromCaption(curveCaption);
+         string curveCaption = captionFor(new[] { seriesFieldValues[STATISTICAL_AGGREGATION_DISPLAY_NAME] });
```

Comment:
Can you comment on these changes? I am not sure I understand what's going on?

Reply:
![image](https://user-images.githubusercontent.com/7632459/173299929-d3ff0f73-1bc5-4f26-b30b-a21e56b313b4.png)

It used to show the name of the compound on the top separator, then it repeated the name of the compound, the observer and the statistical measure on the header of each column. The thing is that the top header is shared for several statistical measures and it makes no sense to repeat then the same data again and again on all columns if they all share the common top header. So I removed al repeated data from the second header (in red in the image) and only left what makes it unique. The compound is already on the top header (in yellow in the image). Currently we do not show the name of the observer but maybe could be inferred from the compound, otherwise I would add it too but to the top header only. This way the view is more compact and does not repeat the same information again and again (which at the end you only see if you roll over it since it does not fit in the header). If you think this is not a good idea we could discuss it and we could restore it to the way it was before.

Reply:
I am tagging @PavelBal since he could also give some useful insights here.

Reply:
Actually, the caption is used to draw the chart where we will not have the table representation so we need to keep the naming as it was before. I am reverting this changes now.